packages/block-editor/src/store/selectors.js

```diff
@@ -2737,8 +2731,7 @@ export const __unstableGetContentLockingParent = createSelector(
 	( state, clientId ) => {
 		let current = clientId;
 		let result;
-		while ( state.blocks.parents.has( current ) ) {
-			current = state.blocks.parents.get( current );
+		while ( ( current = current = state.blocks.parents.get( current ) ) ) {
 			if (
 				( current &&
```

Comment:
This is strange. There is a typo in `current = current = ...`, but also why is there this `if ( current )` check, if the `while` condition already ruled it out?

It's almost like `null` or `undefined` were valid values in the map. Then `has( current )` is true, but `get( current )` is falsy.

Reply:
Whoops, that's a typo!